component/src/dex/tests.rs

```diff
@@ -29,6 +29,13 @@ mod test {
         let height = 1;
 
         state_tx.put_block_height(height);
+        state_tx.put_epoch_by_height(
+            height,
+            penumbra_chain::Epoch {
+                index: 0,
+                start_height: 0,
+            },
+        );
```

Comment:
Do we ever want to set the block height without also setting the epoch? If not, should we fold both of these methods into a `put_block_height_and_epoch` method?

Reply:
In `init_chain`, we want to set the epoch for both the current block (0) and the first block (1), since we only set the epoch in `end_block`, and `end_block` isn't called after init chain on block 0